src/tile/index.html

```diff
@@ -0,0 +1,12 @@
+<div class="gb-product-tile">
+  <a href="{ $product.link() }" onclick="{ $product.onClick }"><img riot-src="{ $product.data.image }" alt="" class="gb-product-tile__image"></a>
+  <div class="gb-product-tile__content">
+    <h3 class="gb-product-tile__content__title"><a href="{ $product.link() }" onclick="{ $product.onClick }">{ $product.data.title }</a></h3>
+    <span class="gb-product-tile__content__price">{ $product.data.price }</span>
```

Comment:
`gb-product-tile__content` isn't a block, it's an element
and there's no reason that `gb-product-tile__price` has to belong to content is there?
I think we want to avoid having multiple element levels or multiple modifiers in the class name, I don't think that's the BEM way